packages/devtools_app/lib/src/screens/memory/memory_controller.dart

```diff
@@ -40,6 +40,34 @@ class OfflineFileException implements Exception {
   String toString() => message;
 }
 
+class MemoryChildControllers {
```

Comment:
rename to MemoryFeatureControllers to match naming in other screens.

It would be worth investigating state management between the performance page and the memory page to see if we should use a common approach there. The Performance page and the Memory page have a similar setup where each tab contains a somewhat self contained feature. In the performance screen, we have a concept of "activeFeature" and each feature controller extends `PerformanceFeatureController`, which is an abstract class with lifecycle methods that each feature controller should implement (some examples `init`, `onBecomingActive`, `setOfflineData`, `clearData`, etc.). This setup is nice from an organization standpoint but also prevents us from doing business logic work in a controller that we don't need to be doing if a feature is not active. Not pressing to do in this PR, but just food for thought.

Reply:
thanks

Noted this for the time I will work to support offline mode for memory screen.